sources/iTermAdvancedSettingsModel.m

```diff
@@ -442,9 +442,9 @@ + (void)enumerateDictionaries:(void (^)(NSDictionary *))block {
 DEFINE_BOOL(neverWarnAboutPossibleOverrides, NO, SECTION_WARNINGS @"Suppress a warning about a change to a global key that's overridden by a Profile.");
 DEFINE_BOOL(noSyncNeverRemindPrefsChangesLostForUrl, NO, SECTION_WARNINGS @"Suppress changed-setting warning when prefs are loaded from a URL.");
 DEFINE_BOOL(noSyncNeverRemindPrefsChangesLostForFile, NO, SECTION_WARNINGS @"Suppress changed-setting warning when prefs are loaded from a custom folder.");
-DEFINE_BOOL(noSyncSuppressAnnyoingBellOffer, NO, SECTION_WARNINGS @"Suppress offer to silence bell when it rings too much.");
+DEFINE_BOOL(noSyncSuppressAnnoyingBellOffer, NO, SECTION_WARNINGS @"Suppress offer to silence bell when it rings too much.");
```

Comment:
We can't change the first parameter of the DEFINE_XXX() macros because it would reset people's preferences.